### ISSUE TITLE
Missing labels for checkboxes on Manage Embargoes page

### ISSUE THREAD

**Issue by rjkati** [samvera/hyrax#6783]
### Descriptive summary

On dev.nurax.samvera.org (Hyrax 5.0.1), the checkboxes on the Manage Embargoes page do not have labels and therefore do not meet WCAG AA requirements

### Steps to reproduce the behavior in User Interface (UI)

1. In Chrome, download the Site Improve browser extension
2. Go to https://dev.nurax.samvera.org/embargoes?locale=en#expired and run the Site Improve browser extension

![form_field_label](https://github.com/samvera/hyrax/assets/32644254/ccc9a779-ce8f-48e6-83c9-d19d510bd698)

### Related work
#6778

**Comment by christhepianist** [comment]
Tested and confirmed.

**Comment by christhepianist** [comment]
All of the form fields have labels except those that are filesets without titles.![Image](https://github.com/user-attachments/assets/555d495f-c18d-491b-a48f-0e7ebd747583)

@jlhardes I think you are right. There is a fileset with a title that is not throwing an exception, but all of them without titles are. I don't think this is your problem.

**Comment by aahurford** [comment]
Fixed in dev, still present in pg:
![Image](https://github.com/user-attachments/assets/478471a6-605c-4f14-9c91-4eee376022d5)

**Comment by christhepianist** [comment]
@jlhardes Most of these checkboxes are fine now, but 6 of them still show that they are missing a label: the 1st, 3rd, 7th, 17th, 19th and 22nd.
